lit_nlp/client/elements/table.ts

```diff
@@ -105,6 +116,11 @@ export class DataTable extends ReactiveElement {
     this.reactImmediately(() => this.rowFilteredData, filteredData => {
       this.stickySortedData = null;
     });
+
+    // TODO(lit-dev) not sure why this isn't working on its own
+    this.sortName = this.defaultSortName;
+    this.sortAscending = this.defaultSortAscending;
+    this.requestUpdate();
```

Comment:
@jameswex @iftenney `sortName` seems like it was private to me, so I couldn't figure out how to pass it.  I can look at this again, but it sounds like maybe I was misunderstanding something here, so let me know if you have any thoughts!